### ISSUE TITLE
Update Registration to cater for disabilities

### ISSUE THREAD

**Issue by avishkar58** [deep-learning-indaba/Baobab#301]
TODO: More details
The registration form needs a question along the lines of: 
"You indicated in your user profile that you have a <user_category.disability> disability, please provide more details on how we can best support you at the Indaba" that is shown if the user profile has a disability selected.

**Comment by jaderabbit** [comment]
@avishkar58 I'd imagine we'd add a "show_when_disabled" field in registration? Work similarly to the others?

**Comment by avishkar58** [comment]
Lets keep this low-priority for now - I don't think there will be many candidates affected by this, so we could also just contact them individually.

**Comment by avishkar58** [comment]
No longer relevant as we're not capturing disabilities in the user profile anymore after #439
